packages/cli/src/services/pruning/pruning.service.ts

```diff
@@ -170,16 +144,13 @@ export class PruningService {
 		} catch (error) {
 			this.logger.error('Failed to hard-delete executions', {
 				executionIds,
-				error: error instanceof Error ? error.message : `${error}`,
+				error: ensureError(error),
 			});
 		}
 
-		/**
-		 * For next batch, speed up hard-deletion cycle in high-volume case
-		 * to prevent high concurrency from causing duplicate deletions.
-		 */
-		const isHighVolume = executionIds.length >= this.hardDeletionBatchSize;
+		// if high volume, speed up next hard-deletion
+		if (executionIds.length >= this.batchSize) return 1_000;
```

Comment:
To be explicit that we are retuning milliseconds

```suggestion
		if (executionIds.length >= this.batchSize) return 1 * Time.seconds.toMilliseconds;
```